ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -380,6 +380,7 @@
         :hintText (or hint-text (placeholder field data))
         :disabled disabled?
         :value chips
+        :data-cypress data-attribute-cypress
```

Comment:
What if data-attribute-cypress is not given? It might be better to add this attribute only if it is given in form)